style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+  font-family: sans-serif;
+  margin: 0rem;
+}
+
+nav {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+a, a:visited, a:active {
+  text-decoration: none;
+  color: white;
+  padding: 0.5rem;
+}
+
+p {
+  margin: 0.5rem;
+}
+
+.top-nav {
+  width: 100%;
+  border: solid 1px #006064;
+  background: #006064;
+  overflow: auto;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+}
+
+.nav-list_item {
+  margin: 0.5rem 0.5rem;
+  text-transform: uppercase;
+  background: gray;
+  padding: 0.5rem;
+  border-radius: 5px;
+  text-decoration: none;
+}
+
+.nav-list_item--brand {
+  flex: 1;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 0.5rem;
+}
+
+.nav-list_item:hover {
+  background: lightgrey;
+}
+
+.body-img {
+  display: inline-block;
```

Comment:
I don't think you need `inline-block` here. That might eventually lead to unwanted side effects. Since the picture is on a new line each time in the design, then use `block`.